### ISSUE TITLE
If without else. and matches that don't cover all constructors.  These are okay provided they end in a return statement.

### ISSUE THREAD

**Issue by PaulBone** [PlasmaLang/plasma#123]

**Comment by PaulBone** [comment]
Duplicate of #122

**Comment by PaulBone** [comment]
Actually it just depends on #122..  #122 enables early return and this bug relaxes some syntax requrements once early return/throw is available.
